mod/missions/views/default/forms/missions/sort-missions-form.php

```diff
@@ -141,4 +142,39 @@
         </div>
 		
 	</div>
+    
+    <div class="col-sm-11 mrgn-tp-lg">
+        <strong><?php echo elgg_echo('missions:filter'); ?></strong>
```

Comment:
If there are no current filters can we hide this and only have it display when there are filters.